interview-questions.md

```diff
@@ -8,32 +8,34 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the virtual DOM in React?
 
-Your answer:
+Your answer: The virtual DOM is basically the threshold between the DOM and React.
 
-Researched answer:
+Researched answer: The virtual DOM is a concept in React where the virtual representation of the UI is kept in memory, not the actual DOM that React maintains.
 
 2. Ruby has an implicit return. What does that mean?
 
-Your answer:
+Your answer: In ruby, methods have implicit return. This basically means that they will be automatically returned without having to "call" upon the return keyword.
 
-Researched answer:
+Researched answer: This implicit return feature in Ruby makes code more concise and readable, as you don't need to use return explicitly unless you want to exit the method or block early or return a specific value.
```

Comment:
it's important to note that Ruby will return the last line of code before the end of the method with implicit return